remote-config.jsonc

```
@@ -252,16 +246,31 @@
           "message": "`ddev aliases` will show you the alternate versions of commands you can use. For example, `ddev st` for `ddev describe`."
         },
         {
-          "message": "DDEV v1.24.2+ has MySQL 8.4 and there's lots of database performance improvements, see https://ddev.com/blog/database-improvements"
+          "message": "DDEV has MySQL 8.4 and there's lots of database performance improvements, see https://ddev.com/blog/database-improvements"
         },
         {
           "message": "Take the Open Source Pledge and include DDEV in your plans based on its value to your organization, https://ddev.com/blog/open-source-pledge"
         },
         {
-          "message": "Are you interested in how 14,000 developers are using DDEV weekly and what they're doing with it? See the latest analysis, https://ddev.com/blog/stats-on-ddev-usage-nov-2024"
+          "message": "Are you interested in how 17,000 weekly developer users are using DDEV and what they're doing with it? See the latest analysis and live charts, https://ddev.com/blog/stats-on-ddev-usage-nov-2024"
+        },
+        {
+          "message": "DDEV supports ARM64 Windows, especially under WSL2, but does great everywhere, macOS (native ARM64), Linux, Windows - https://ddev.com/blog/windows-ddev-setup"
+        },
+        {
+          "message": "DDEV v1.24.7 will have MariaDB 11.8 LTS, with its new vector support"
+        },
+        {
+          "message": "DDEV v1.24.7 will have modest support for Rancher Desktop on Windows (for traditional Windows) and Docker Desktop for Linux"
+        },
+        {
+          "message": "Understand DDEV's commitment to financial and community sustainability: https://ddev.com/blog/sustainability-for-ddev/"
+        },
+        {
+          "message": "DDEV can even spin up ancient PHP projects with ancient database versions, even on Apple Silicon: https://ddev.com/blog/legacy-projects-with-unsupported-php-and-mysql-using-ddev"
         },
         {
-          "message": "DDEV supports ARM64 Windows very nicely, especially under WSL2, but does great everywhere, macOS (native ARM64), Linux, Windows - https://ddev.com/blog/windows-ddev-setup"
+          "message": "Maintaining DDEV add-ons? See latest recommended best practices at https://ddev.com/blog/ddev-add-on-maintenance-guide"
```

```suggestion
          "message": "Maintaining a DDEV add-on? See latest recommended best practices at https://ddev.com/blog/ddev-add-on-maintenance-guide"
```
Wouldnt it better to use the singular form here? everyone starts with their first addon ;)